taxi/views.py

```diff
@@ -1,3 +1,18 @@
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
 
+from taxi.models import Driver, Manufacturer, Car
+
+
 # Create your views here.
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.all().count()
+    num_manufacturers = Manufacturer.objects.all().count()
+    num_cars = Car.objects.all().count()
+    context = {
```

Comment:
same here, you can use this dict directly in the render function, no reason to create `context` variable.